playbooks/phobos_vpn.yml

```diff
@@ -1,5 +1,5 @@
-- name: Configure Phobos VPN Tunnel (should only be run on single use slaves)
-  hosts: localhost
+- name: Configure Phobos VPN Tunnel
+  hosts: {{ target_hosts | default(‘localhost’) }}
```

Comment:
I think this needs to be quoted, ie:

hosts: "{{ target_hosts | default(‘localhost’) }}"

Reply:
LOL, will fix. I believed I copied this handy fragment from a Zoom chat session with you.